ghost/core/core/frontend/meta/utils.js

```
@@ -0,0 +1,33 @@
+const security = require('@tryghost/security');
+const debug = require('@tryghost/debug')('frontend');
+const path = require('path');
```

ESLint

[error] 'path' is assigned a value but never used.

(no-unused-vars)